cli/js/lib.deno.ns.d.ts

```diff
@@ -2134,19 +2134,20 @@ declare namespace Deno {
    */
   export function metrics(): Metrics;
 
-  /** **UNSTABLE**: reconsider representation. */
   interface ResourceMap {
-    [rid: number]: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [rid: number]: any;
```

Comment:
It would be useful to have a comment here explaining why we’re using any instead of string.

Reply:
PTAL at docstring for resources()